### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by lauriefish22**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -14,19 +14,51 @@ import Signup from './pages/Signup';
 import UserProfile from './pages/UserProfile.js';
 import About from './pages/About';
 import Detail from './pages/Detail';
+<<<<<<< HEAD
```
Comment (aarticontractor) [inline client/src/App.js:17]: can you fix conflcts before we merge

File: client/src/App.js [hunk 2]
```diff
@@ -14,19 +14,51 @@ import Signup from './pages/Signup';
...
 import { StoreProvider } from './utils/globalState';
 
 
 
+>>>>>>> main
 import NavBar from './components/NavBar';
 import Home from './pages/Home';
 // import { DefaultDeserializer } from 'v8';
 
+<<<<<<< HEAD
```
Comment (aarticontractor) [inline client/src/App.js:37]: here as well

**Review by Berkeleycodingmomma — APPROVED** [review]
donion rings

File: client/src/App.js [hunk 3]
```diff
@@ -61,6 +93,10 @@ function App() {
                             <Route path='/products/:id' element={<Detail />} />
                             <Route path='/UserProfile' element={<UserProfile />} />
                         </Routes>
+<<<<<<< HEAD
```
Comment (aarticontractor) [inline client/src/App.js:96]: here as well

**Review by aarticontractor — CHANGES_REQUESTED** [review]

**Pull request merged**